cafebabel/templates/articles/detail.html

```diff
@@ -1,10 +1,31 @@
 {% extends "base.html" %}
+{% from "_meta-macro.html" import meta without context %}
 {% import "articles/macros.html" as macros without context %}
 
 {% block meta_title %}{{ article.title }}{% endblock %}
-{% block meta_description %}{{ article.summary }}{% endblock %}
+{% block metas %}
+  {{ meta('name=description', article.summary) }}
+  {{ meta('name=author', article.author) }}
+{% endblock %}
+<!--Social-->
```

Comment:
Not sure that comment is still relevant given how explicit is the block name 👍

Reply:
fixed